cpp/src/arrow/util/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ add_arrow_test(utility-test
                SOURCES
                align_util_test.cc
                atfork_test.cc
+               bitmap_test.cc
```

Comment:
Would rather add this to `bit-utility-test` than to the already overcrowded `utility-test`.

Reply:
Yes, sounds right. Didn't see `bit-utility-test` is actually a collection of several test files. Will do.